pkg/cmd/openfaas_app.go

```diff
@@ -246,6 +253,17 @@ func getValuesSuffix(arch string) string {
 	return valuesSuffix
 }
 
+func mergeFlags(existingMap map[string]string, customs []string) error {
```

Comment:
Please can you add a unit test for this method in `openfaas_app_test.go`? 

For reference: https://blog.alexellis.io/golang-writing-unit-tests/